src/js/edu-benefits/pages/applicantInformation.js

```diff
@@ -51,7 +51,14 @@ export default function applicantInformation(schema, options) {
       'ui:order': fields,
       relativeFullName: fullName.uiSchema,
       relativeSocialSecurityNumber: ssn.uiSchema,
-      relativeDateOfBirth: currentOrPastDate.uiSchema('Date of birth'),
+      relativeDateOfBirth: _.assign(currentOrPastDate.uiSchema('Date of birth'),
+        {
+          'ui:errorMessages': {
+            pattern: 'Please provide a valid date',
+            futureDate: 'Please provide a valid date'
```

Comment:
I'm not too fond of this, but should I override the `pattern` and use that in the validation? `futureDate` is specifically for if the date is not in the future, whereas `pattern` is for if it's not a valid date.